izbor.cpp

```diff
@@ -46,67 +59,86 @@ void Izbor::ucitajKviz()
+        ui->comboBox_odgovori->addItem(kviz.getKolekcijaPitanja()[i].getOdgovor(j));
     }
-    ui->comboBox_odgovori->setPlaceholderText("--Vas odgovor--");
-    ui->comboBox_odgovori->setCurrentIndex(-1);
+    ui->comboBox_odgovori->setCurrentIndex(odg[i] - 1);
 }
 
-void Izbor::ucitajIme(QString ime)
-{
-    imeIgraca = ime;
-}
-
-void Izbor::on_pushButton_potvrdaOdg_clicked()
+void Izbor::ispis()
 {
-    if(ui->comboBox_odgovori->currentIndex() != -1)
+    for(int k = 0; k < 7; k++)
     {
-        odg = ui->comboBox_odgovori->currentIndex() + 1;
-        qDebug() << odg;
-        if(odg == kviz.getKolekcijaPitanja()[i].getTacanOdgovor())
+        if(odg[k] == kviz.getKolekcijaPitanja()[k].getTacanOdgovor())
         {
             tacni_odg++;
-            qDebug() << "Br tacnih odg: " << tacni_odg;
         }
+    }
 
-        i++;
+    ui->label_Pitanje->setText("Takmicar " + imeIgraca + " je pogodio " + QString::number(tacni_odg) + "/"
+                               + QString::number(kviz.getKolekcijaPitanja().size()) + " pitanja.");
+    ui->label_brPitanja->setVisible(false);
+    ui->comboBox_odgovori->setVisible(false);
+    ui->pushButton_naredno->setVisible(false);
+    ui->pushButton_prethodno->setVisible(false);
+    ui->pushButton_zavrsi->setVisible(false);
+    QPixmap slika3("://database/Purple-Confetti");
+    w = ui->label_konfete->width();
+    h = ui->label_konfete->height();
+    ui->label_konfete->setPixmap(slika3.scaled(w, h, Qt::IgnoreAspectRatio));
+}
 
-        if(i < 7)
-        {
-            ui->comboBox_odgovori->clear();
-            ui->label_brPitanja->setText("Pitanje " + QString::number(i + 1) + ":");
-            ui->label_Pitanje->setText(kviz.getKolekcijaPitanja()[i].getPitanje());
-            for (int j = 0; j < 4; j++)
-            {
-                ui->comboBox_odgovori->addItem(kviz.getKolekcijaPitanja()[i].getOdgovor(j));
-            }
-
-            odg = ui->comboBox_odgovori->currentIndex() + 1;
-            qDebug() << odg;
-            if(odg == kviz.getKolekcijaPitanja()[i].getTacanOdgovor())
-            {
-                tacni_odg++;
-                qDebug() << "tacni_odg";
-            }
-        }
+void Izbor::on_pushButton_naredno_clicked()
+{
+    odg[i] = ui->comboBox_odgovori->currentIndex() + 1;
 
-        if(i == 7)
-        {
-            ui->label_Pitanje->setText(imeIgraca + " je pogodio " + QString::number(tacni_odg) + "/"
-                                       + QString::number(kviz.getKolekcijaPitanja().size()) + " pitanja.");
-            ui->label_brPitanja->setVisible(false);
-            ui->comboBox_odgovori->setVisible(false);
-            ui->pushButton_potvrdaOdg->setVisible(false);
-            QPixmap slika3("://database/Purple-Confetti");
-            w = ui->label_konfete->width();
-            h = ui->label_konfete->height();
-            ui->label_konfete->setPixmap(slika3.scaled(w, h, Qt::IgnoreAspectRatio));
-        }
-    }
+    i++;
```

Comment:
Možda bi ovo ``i`` moglo da bude malo deskriptivnije nazvano (indeks trenutnog pitanja ili nešto slično).